drivers/sensor/tmp112/Kconfig

```diff
@@ -12,3 +12,42 @@ config TMP112
 
 	  The TMP102 is compatible with the TMP112 but is less accurate and has
 	  been successfully tested with this driver.
+
+choice
+	prompt "Conversion rate"
+	default TMP112_CR_RUNTIME
+	help
+	Initial conversion rate set at initialization
+	Supported values 0.25Hz, 1Hz, 4Hz, 8Hz
+
+config TMP112_CR_RUNTIME
+	bool "Set at runtime"
+
+config TMP112_CR_025HZ
+	bool "0.25Hz"
+
+config TMP112_CR_1HZ
+	bool "1Hz"
+
+config TMP112_CR_4HZ
+	bool "4Hz"
+
+config TMP112_CR_8HZ
+	bool "8Hz"
+endchoice
```

Comment:
This setting should be part of devicetree, not Kconfig

Reply:
True this would allow to have per device init values

Reply:
Of course in DT to allow multi-instance